src/ui/Root.fs

```diff
@@ -1,17 +1,42 @@
 module Store.UI.Root
 
+open System
+
+open Browser
+open Browser.Navigator
 open Feliz
 open Feliz.Router
+open Store.Shared.DomainModel
+open Store.UI.About
 open Store.UI.Error
 open Store.UI.Home
-open Store.UI.About
-open Browser

```

Comment:
Why were these 2 lines removed?

Reply:
They weren't. They were moved up and sorted. :)